src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -11910,6 +11910,32 @@ class C { }
 
             // https://github.com/dotnet/roslyn/issues/48765 tracks enabling support for this scenario.
             CreateCompilation(source).VerifyEmitDiagnostics(
+                // (12,14): error CS0214: Pointers and fixed size buffers may only be used in an unsafe context
```

Comment:
>// (12,14): error CS0214: Pointers and fixed size buffers may only be used in an unsafe context [](http://example.com/codeflow?start=16&length=95)

Is this error language version dependent? We probably should test the fact:
- version 11
- version 12
- preview

Reply:
I see this tested in AttributeTests.cs